src/etc/rc.initial.toggle_sshd

```diff
@@ -31,6 +31,12 @@ require_once("filter.inc");
 $fp = fopen('php://stdin', 'r');
 
 if (isset($config['system']['ssh']['enable'])) {
+	if (!empty(getenv('SSH_CONNECTION'))) {
+		echo (gettext("!!! WARNING !!!"));
+		echo "\n\n";
+		echo (gettext("This session is currently established via SSH. Disabling SSH will terminate this session and prevent future SSH connections from any user. Before disabling SSH, ensure that administrators have alternate means of accessing the firewall, such as the GUI or serial console."));
```

Comment:
No need for the extra `()` around the echo contents.

Reply:
Gotcha. They're gone! Is there a style guide for the project?

Reply:
Yep, there is https://docs.netgate.com/pfsense/en/latest/references/developer-style-guide.html (for development) and also https://docs.netgate.com/pfsense/en/latest/references/style-guide.html for documentation (also should apply to text in the GUI/menus/etc where possible)

Reply:
Nice! Thanks